drivers/clock_control/clock_control_mcux_syscon.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-23, NXP
+ * Copyright 2020-24 NXP
```

Comment:
missed this before

Reply:
This looks fine, am I missing something?

Reply:
it should be 2020-2024